src/wix/WixToolset.Core.Burn/Bundles/BurnReader.cs

```diff
@@ -168,16 +170,19 @@ public bool ExtractAttachedContainer(string outputDirectory, string tempDirector
             }
 
             Directory.CreateDirectory(outputDirectory);
-            string tempCabPath = Path.Combine(tempDirectory, "attached.cab");
-
-            this.binaryReader.BaseStream.Seek(this.AttachedContainerAddress, SeekOrigin.Begin);
-            using (Stream tempCab = File.Open(tempCabPath, FileMode.Create, FileAccess.Write))
+            foreach (ContainerSlot cntnr in this.AttachedContainers)
             {
-                BurnCommon.CopyStream(this.binaryReader.BaseStream, tempCab, (int)this.AttachedContainerSize);
-            }
+                string tempCabPath = Path.GetTempFileName();
```

Comment:
Should bring back the directory parameter because the Core integration tests won't be able to delete this file.